### DISCUSSION TITLE
V3 Feedback and Suggestions

### DISCUSSION THREAD

**Discussion by sdaqo** [sdaqo/anipy-cli#160] [Q&A]
So I never used discussions, but I thought this is a good use case! 
Please, if you have some, you can post (feature) suggestions  or rant about stuff that annoys you in the new v3 version, be it about the cli or the api!
Any kind of answers are appreciated!

**Comment by sdaqo** [comment]
Some stuff that is already planned:
- A local provider so you can search and watch your local anime through anipy-cli
- Maybe a Jellyfin provider for easy cli access to jellyfin

**Comment by Quicksilver151** [comment]
Not much complaints really. I've never used anipy as a library so I doubt it makes much difference to me. That said, I'm still on version 2. I have yet to learn how to use version 3
